tabs/src/components/app/views/DocumentationLinks.jsx

```diff
@@ -1,28 +1,46 @@
 import React from "react";
 import { UNLOCALIZED_GRAPH_EXPLORER_URL, UNLOCALIZED_GRAPH_EXPLORER_DOCS_URL, UNLOCALIZED_OFFICIAL_RSC_URL, localizeUrl } from "../../TabConstants";
-import { Button } from '@fluentui/react-northstar';
 import { useTranslation } from 'react-i18next';
+import { useTeamsFx } from "../../utils/useTeamsFx";
 
 export const DocumentationLinks = () => {
     const { t, i18n } = useTranslation();
+    const { themeString } = useTeamsFx();
+
+    let linkColor = "#6264A7";
+    switch(themeString) {
+        case "default":
+            linkColor = "#6264A7";
+            break;
+        case "dark":
+            linkColor = "#A6A7DC";
+            break;
+        case "contrast":
+            linkColor = "#ffff01";
+            break;
+        default:
+            linkColor = "#6264A7";
+    }
 
     return (
         <div className="documentation-links">
-            <Button
-                content={t("Documentation Links.Go to Graph Explorer")}
-                onClick={() => window.open(localizeUrl(UNLOCALIZED_GRAPH_EXPLORER_URL, i18n))}
-                primary
-            />
-            <Button
-                content={t("Documentation Links.Graph Explorer Documentation")}
-                onClick={() => window.open(localizeUrl(UNLOCALIZED_GRAPH_EXPLORER_DOCS_URL, i18n))}
-                primary
-            />
-            <Button
-                content={t("Documentation Links.Resource-Specific Consent")}
-                onClick={() => window.open(localizeUrl(UNLOCALIZED_OFFICIAL_RSC_URL, i18n))}
-                primary
-            />
+            <ul>
+                <li>
+                    <a className="documentation-link" href={localizeUrl(UNLOCALIZED_GRAPH_EXPLORER_URL, i18n)} target="_blank" rel="noreferrer noopener" style={{ color: linkColor }}>
```

Comment:
lots of duplication, any way to make a Link component?

Reply:
114882381f95ea17277872d3207708424e29c5ea